docs/diagrams/contactTaskTag/ParserClassDiagram.puml

```diff
@@ -0,0 +1,12 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
```

Comment:
Since parser is part of the logic component, perhaps change the color to LOGIC_COLOR